requirements-test.txt

```diff
@@ -14,6 +14,7 @@ peft
 requests
 ray
 sentence-transformers # required for embedding
+torchvision==0.18.0 # required for phi3v
```

Comment:
Do we need such a strict version requirement?

Reply:
This is required by the image processor of phi3v. You can check out their image processor here: [image_processing_phi3_v.py](https://huggingface.co/microsoft/Phi-3-vision-128k-instruct/blob/main/image_processing_phi3_v.py)

Reply:
I see, can you mention this in the requirements file to avoid confusion? Thanks!